examples/incomplete_async_payloads/src/lib.rs

```diff
@@ -1,11 +1,10 @@
 #![no_std]
 
 use gstd::{debug, msg, prelude::*};
-use gstd_async::msg as msg_async;
 
 const GAS_LIMIT: u64 = 50_000_000;
 
-#[gstd_async::main]
+#[gstd::main]
```

Comment:
macros name or path should have something about async

Reply:
What about a small refactor of macro under the hood, which will be no-op in case of non-async function?

Reply:
Please, no

Things should be explicit

If no macro required, then no macro should be used

Reply:
May to rename it to `#[gstd::async_main]`?

Reply:
What about universal `#[gstd::async]`, which one we gonna use for handle and init in future?

Reply:
I don't like underscores in macros

Reply:
I would be good if it fits habitual patterns like  tokio::main etc. But I unknown the best name for our case 😄